controllers/user/main.go

```diff
@@ -52,15 +52,20 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
+	var (
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+		concurrent           int
+		rateLimiterOptions   utilcontroller.RateLimiterOptions
+	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-
+	flag.IntVar(&concurrent, "concurrent", 5, "The number of concurrent cluster reconciles.")
```

Comment:
also need to add this parameter in config: `concurrent, min-retry-delay, max-retry-delay`

Reply:
rateLimiterOptions.BindFlags(flag.CommandLine)

already exsti this config